runtime/publish.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+# TODO dynamically change the region
+REGION=us-east-2
+BREF_VERSION=0.3
+PHP_VERSION=7.2.5
```

Comment:
any chance to get a more recent php version?

Reply:
Oh yeah we just have to change that line. However I thought it would be fair to compare to the same version used in Bref currently for the benchmarks.

Let's keep that comment open so that I remember to update this line before merging.